recipe/meta.yaml

```diff
@@ -53,7 +53,6 @@ requirements:
 
 test:
   commands:
-    - python -c "import pkg_resources; pkg_resources.require('Orange3')"
```

Comment:
Not sure what error are you trying to fix by removing this test.
Its purpose is to check that all requirements specified in setup.py (and their requirements, and so on recursively) are met after the package is installed via conda.

Reply:
just trying to shake things to see where the problem is....

I think we are stumbling into some weird anaconda-python bugs..